SendToFactorio.ps1

```diff
@@ -6,10 +6,8 @@ $DIRS = @(
 ".\omnilib",
 ".\omnimatter",
 ".\omnimatter_compression",
-".\omnimatter_crystal",
 ".\omnimatter_permutation",
-".\omnimatter_water",
-".\omnimatter_wood"
+".\omnimatter_water"
```

Comment:
Wood was removed because the game doesn't launch with it active, or at least it was in that state when I was working on it. I'm fine removing this file and adding to .gitignore.

Reply:
The file itself can stay, but up on the repo it should include all released omnimods.